arduino-ide-extension/src/browser/style/list-widget.css

```diff
@@ -62,6 +63,13 @@
     filter: contrast(90%);
 }
 
+.filterable-list-container .no-match {
+    font-weight: bold;
+    color: var(--theia-panelTitle-inactiveForeground);
+    align-items: center;
+    margin: 10px;
+}
+
 .component-list-item {
```

Comment:
It must come from a manual merge; it's obsolete. Please remove. Thanks!

Reply:
Removed [here](https://github.com/arduino/arduino-ide/pull/1361/commits/88b0a9371beecca44293bd6255ae52057ec05ae8).

Reply:
Done.